internal/rbd/rbd_util.go

```diff
@@ -621,6 +631,28 @@ func genVolFromVolumeOptions(ctx context.Context, volOptions, credentials map[st
 	return rbdVol, nil
 }
 
+func validateImageFeatures(imageFeatures, mounter string) error {
+	arr := strings.Split(imageFeatures, ",")
+	featureSet := sets.NewString(arr...)
+	for _, f := range arr {
+		sf, found := supportedFeatures[f]
+		if !found {
+			return errors.Errorf("invalid feature %s for volume csi-rbdplugin", f)
```

Comment:
```suggestion
return errors.Errorf("invalid feature %s for %s", f,reqName)
```
please pass  request name for more  clarity